api/ocm/extensions/blobhandler/config/type.go

```diff
@@ -77,16 +79,16 @@ func (a *Config) ApplyTo(ctx cfgcpi.Context, target interface{}) error {
 
 const usage = `
 The config type <code>` + ConfigType + `</code> can be used to define a list
-of preconfigured download handler registrations (see <CMD>ocm ocm-downloadhandlers</CMD>):
+of preconfigured upload handler registrations (see <CMD>ocm ocm-uploadhandlers</CMD>):
 
 <pre>
     type: ` + ConfigType + `
-    description: "my standard download handler configuration"
+    description: "my standard upload handler configuration"
     handlers:
-      - name: oci/artifact
-        artifactType: ociImage
-        mimeType:
-        config: ...
+      - name: ` + ocirepo.BlobHandlerName + `
```

Comment:
lets change to builtin/... as discussed?

Reply:
no, see below